bo/bo_loop.py

```diff
@@ -36,7 +39,29 @@ def __init__(self, black_box_func: BaseTestProblem,
         self.performance_type = performance_type
         self.acquisition_function_type = ei_type
 
+    def save_parameters(self):
+
+        self.unix_time = round(time.time())
+        print(f'Current time: {self.unix_time}')
+
+        # Create folder for saving data.
+        folder_path = os.getcwd()
+        self.folder = os.path.join(folder_path, str(self.unix_time))
+        os.mkdir(self.folder)
+
+        # TODO: add more parameters
+        parameters = {
+            'seed': self.seed,
+            'budget': self.budget,
```

Comment:
We need to save wayy more params. Data collected so far (all inputs, function values and constraint values). Best location and value according to the posterior mean at each iteration. Best location and value according to the data sampled so far at each iteration. Hyperparameters of the gaussian process at every iteration, i.e, lengthscales and variance.